docker/docker-compose.yml

```diff
@@ -47,20 +47,11 @@ services:
     build:
       <<: *common-build
       target: worker
-  worker_exposure_payload:
+  worker_scheduled:
```

Comment:
Is it all scheduled?

Reply:
What do you mean? 🤔

Reply:
Whether all the tasks that run within this worker are scheduled? (I guess so, just double-checking.)

Reply:
Ah, yes they are : D